src/test/java/lambda/part1/exercise/Lambdas01Exercise.java

```diff
@@ -21,7 +24,12 @@ public void sortPersonsByAge() {
                 new Person("name 2", "lastName 1", 30)
         };
 
-        // TODO use Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person left, Person right) {
+                return left.getAge() - right.getAge();

```

Comment:
In general it's incorrect. You can't compare `ints` this way. Use `Integer.compare`